File: Practice/13/C++/13/13/13.cpp
#include <iostream>
#include <math.h>
using namespace std;
int main()
{   setlocale(LC_ALL, "Russian");
    int n;
    bool flag = true;
    cin >> n;
    if (n >= 2) {
        for (int i = 2; i <= sqrt(n); i++) {
            if (n % i == 0) {
                flag = false;
                break;
            }
        }

        if (flag == true) {
            cout << "Простое" << endl;
        }
        else {
            cout << "Сложное" << endl;
        }
    }
    
}

File: Practice/13/Python/13.py
import math
n = int(input())
i = 2
limit = int(math.sqrt(n))
flag = 1
if n < 2:
    print("Число должно быть больше 2")
else:
    while i <= limit:
        if n % i == 0:
            print("Составное")
            flag = 2

        i = i + 1
    if flag == 1:
        print("Простое")